README.md

```diff
@@ -22,30 +37,56 @@ or
 
 Download archives from the following URLs.
 
-* GROMACS 2022: https://gitlab.com/gromacs/gromacs/-/archive/v2022-rc1/gromacs-v2022-rc1.tar.gz
-* gmxapi 0.3: gmxapi-0.3.0b5.tar.gz
-  (Download from https://pypi.org/project/gmxapi/0.3.0b5/#files or use the [`--pre`](https://pip.pypa.io/en/stable/cli/pip_install/#pre-release-versions) option with `pip`
+* GROMACS 2022: https://gitlab.com/gromacs/gromacs/-/archive/release-2022/gromacs-release-2022.tar.gz
+* gmxapi 0.3: gmxapi-0.3.0b6.tar.gz
+  (Download from https://pypi.org/project/gmxapi/0.3.0b6/#files or use the [`--pre`](https://pip.pypa.io/en/stable/cli/pip_install/#pre-release-versions) option: `pip install --pre gmxapi` (after installing and activating GROMACS 2022)
 * *Optional* Run_brer 2.0 (master branch) https://github.com/kassonlab/run_brer/ or https://github.com/kassonlab/run_brer/archive/master.tar.gz
 * *Optional* Brer-plugin current master https://github.com/kassonlab/brer_plugin or https://github.com/kassonlab/brer_plugin/archive/master.tar.gz
 
 ### Install the software
 
 Refer to installation instructions from the software archive or from the respective project web sites.
```

Comment:
Was the archive just for manuscript review or is that something we are still sharing? Should we put a zenodo link in here? (and replace the Google Drive link above?)